docs/find-duplicate-skus/script.liquid

```diff
@@ -0,0 +1,103 @@
+{% assign mappings = hash %}
+{% assign dups = hash %}
+{% assign cursor = nil %}
+
+{% for i in (0..500) %}
+  {% assign after = i | times: 250 %}
+  {% capture query %}
+    query {
+      productVariants(
+        first: 250
+        after: {{ cursor | json }}
+      ) {
+        pageInfo {
+          hasNextPage
+        }
+        edges {
+          node {
+            id
+            sku
+            inventoryItem {
+              tracked
+            }
+          }
+          cursor
+        }
+      }
+    }
+  {% endcapture %}
+
+  {% assign result = query | shopify %}
+
+  {% if event.preview %}
+    {% capture result_json %}
+      {
+        "data": {
+          "productVariants": {
+            "edges": [
+              {
+                "node": {
+                  "id": "gid://shopify/ProductVariant/1234567890",
+                  "sku": "sku123",
+                  "inventoryItem": {
+                    "tracked": "true"
+                  }
+                }
+              },
+              {
+                "node": {
+                  "id": "gid://shopify/ProductVariant/123456789",
+                  "sku": "sku123",
+                  "inventoryItem": {
+                    "tracked": "true"
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    {% endcapture %}
+
+    {% assign result = result_json | parse_json %}
+  {% endif %}
+  
+  {% assign cursor = result.data.productVariants.edges.last.cursor %}
+  {% assign productVariants = result.data.productVariants.edges | map: "node" %}
+
+  {% for pv in productVariants %}
+    {% if pv.sku == null or pv.sku == empty %}
+      {% continue %}
+    {% endif %}
+
+    {% if options.exclude_inventory_not_tracked__boolean == true %}
+      {% if pv.inventoryItem.tracked == false %}
+        {% continue %}
+      {% endif %}
+    {% endif %}
+
+    {% assign existingPv = mappings[pv.sku] %}
+
+    {% if existingPv %}
+      {% if existingPv != pv.id %}
+        {% assign dups[pv.sku] = true %}
+      {% endif %}
+    {% else %}
+      {% assign mappings[pv.sku] = pv.id %}
+    {% endif %}
+  {% endfor %}
+
+  {% unless result.data.productVariants.pageInfo.hasNextPage %}
+    {% break %}
+  {% endunless %}
+{% endfor %}
+
+{% assign skus = productVariants | map: "sku" %}
+{% assign uniqskus = skus | uniq %}
```

Comment:
Capitalization note - use `uniqSkus` here, since we're camelCasing in this task.